jdaviz/configs/default/plugins/model_fitting/tests/test_plugin.py

```diff
@@ -78,8 +78,8 @@ def test_register_model(specviz_helper, spectrum1d):
     # Test that the parameter uncertainties were updated
     expected_uncertainties = {'slope': 0.00038, 'intercept': 2.67}
```

Comment:
Tests are failing because these expected values `[0.00038, 2.67]` are off from the actual values `[0.00024713035501887576, 1.6780202934939978]`. Is it safe to update these numbers since the process for using uncertainties for fitting has been updated upstream in specutils?

Reply:
Model-fitting results will change when uncertainty weights are considered, for non-uniform weighting. See if the test works in the uniform-weighting case (e.g., uncertainty = 0.1 * data).

Reply:
I would also test the case of non-uniform weighting, but on an example where we know the exact answer.

Reply:
Possible tests--- line with slope =1,  Gaussian with sigma =1 and flux = 1.